src/AppInstallerSharedLib/Public/AppInstallerStrings.h

```diff
@@ -10,6 +10,10 @@
 
 namespace AppInstaller::Utility
 {
+    // Same as std::isspace(char)
+#define AICLI_SPACE_CHARS " \f\n\r\t\v"sv

```

Comment:
```suggestion
static constexpr s_AICLI_SPACE_CHARS " \f\n\r\t\v"sv;
```

Reply:
This define is used for creating the expressions used in the trim functions, which is why I didn't change it and only moved it; 
```
    constexpr std::string_view s_SpaceChars = AICLI_SPACE_CHARS;
    constexpr std::wstring_view s_WideSpaceChars = L"" AICLI_SPACE_CHARS;
```